openfisca_country_template/entities.py

```diff
@@ -1,13 +1,29 @@
 # -*- coding: utf-8 -*-
 
 # This file defines the entities needed by our legislation.
+import textwrap
 
 from openfisca_core.entities import build_entity
 
 Household = build_entity(
     key = "household",
     plural = "households",
     label = u'Household',
+    doc = textwrap.dedent('''
+    A group entity.
+    Contains multiple natural persons with specific roles.
```

Comment:
> A group entity contains one or more individual·s. Each individual in a group entity has a role (e.g. parent or children).

Reply:
Sentence used with `natural person` as it is used in law (like `personne physique` in french).